### DISCUSSION TITLE
🥺🥺开机自启或bat启动！这样就可以配合sunshine在外边随时随地原神启动了！EVD，启动！

### DISCUSSION THREAD

**Discussion by xuhan999** [KtzeAbyss/Easy-Virtual-Display#16] [Ideas]
开机自启、bat静默启动、自定义分辨率、自定义刷新率，命令行调参，球球了！🥺🥺🥺🥺🥺🥺
### sunshine原神，启动！
顺带附上自己实验的sunshine原神启动命令！还好米哈游没有阉割太多Unity启动命令~
【Command】那行写启动命令
cmd /C "原神完整路径\YuanShen.exe" use_mobile_platform -is_cloud 1 -platform_type CLOUD_THIRD_PARTY_MOBILE -popupwindow -borderless  -monitor 2 -sdl_displayindex 1
解释：
cmd /C(启动完成自己关闭命令行窗口)"原神完整路径，右键原神主程序选择【复制文件地址】，注意是主程序不是启动器！双引号千万不能去掉"  手机UI界面附加的启动命令  -窗口化运行 -无边框 -在第二块显示器启动 -按照顺序排列你的显示器
【Working Directory】这行写原神的游戏目录，不是安装器的目录，是主程序的目录
比如下图举例
![屏幕截图 2023-12-20 191503](https://github.com/KtzeAbyss/Easy-Virtual-Display/assets/16046777/9214c8d8-fdf1-4fe6-905c-d74206de05d6)
### 分辨率，修改！
BTW，目前分辨率这块，我使用的手机是2400*1080分辨率，虚拟显示器的分辨率我就设置为2560*1080.
好兄弟们可以用近似的分辨率，原则上是与移动端玩的设备尽可能一样的分辨率，可惜现在无法修改，那就只能用近似的了。
规则是，分辨率【高】与设备保持一致，【宽】尽可能选比移动设备更宽的，不然会出现上下黑边。游戏分辨率设置要与虚拟显示器的分辨率保持一致。

修改分辨率注册表位置在
计算机\HKEY_CURRENT_USER\Software\miHoYo\原神
【Screenmanager Is Fullscreen mode_h一串数字】是否全屏游戏，0为窗口化 1为全屏
【Screenmanager Resolution Height_h一串数字】 窗口高度
【Screenmanager Resolution Width_h一串数字】 窗口宽度
比如我这样就设置为：
全屏：0 高度：1080 宽度：2560
### EVD，命令行启动！
未来如果EVD支持BAT启动，还能命令行修改分辨率等参数，那么就可以先设置多个脚本，不同移动设备分辨率不同，刷新率不同，就启动不同的脚本！（爽啊……
所以，球球惹！PLZ~~~🥺🥺🥺🥺

**Comment by 4fuu** [comment]
分辨率修改可以使用nircmd，`nircmd.exe setdisplay 2412 1080 32`
英伟达面板也可以创建自定义分辨率
